.teamcity/settings.kts

```diff
@@ -422,4 +424,89 @@ object NightlyTests : BuildType({
             }
         }
     }
-})
\ No newline at end of file
+})
+
+object Pixi : Project({
+    name = "Pixi"
+
+    buildType(UpdateDependencies)
+})
+
+object UpdateDependencies : BuildType({
+    name = "Update Dependencies"
+
+    params {
+        param("GT_USER", "deltares-service-account")
+        password("env.GH_TOKEN", "credentialsJSON:ced4991a-a11d-463b-b79a-a657eea89523")
+    }
+
+    vcs {
+        root(AbsoluteId("iMOD6_IMODPython_ImodPython"), "+:. => imod-python")
+
+        cleanCheckout = true
+    }
+
+    steps {
+        powerShell {
+            name = "Download Github CLI"
+            id = "Download_Github_CLI"
+            formatStderrAsError = true
+            scriptMode = script {
+                content = """
+                    Invoke-WebRequest -Uri  https://github.com/cli/cli/releases/download/v2.42.1/gh_2.42.1_windows_amd64.zip -OutFile gh.zip
+                    Expand-Archive gh.zip -DestinationPath gh
+                """.trimIndent()
+            }
+        }
+        powerShell {
+            name = "Update dependencies"
+            id = "Update_dependencies"
+            workingDir = "imod-python"
+            scriptMode = script {
+                content = """
+                    ${'$'}env:Path = '%system.teamcity.build.checkoutDir%\gh\bin;' + ${'$'}env:Path  
+                    
+                    echo "Create update branch"
+                    
+                    git remote set-url origin https://%GT_USER%:%env.GH_TOKEN%@github.com/Deltares/imod-python.git
```

Comment:
Why is that necessary?

Reply:
The credentials used during checkout aren't strored. This is by design by TeamCity for security purposes.
Therefor we have to reapply the user and token to be able to commit to the branch